modules/programs/fff.nix

```
@@ -0,0 +1,594 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.programs.fff;
+
+in {
+  meta.maintainers = [ maintainers.gigahawk ];
+
+  options.programs.fff = {
+    enable = mkEnableOption "A simple file manager written in bash";
+
+    cdOnExit = {
+      enable = mkOption {
+        type = types.bool;
+        default = true;
+        description = ''
+          Change directory to current folder on exit
+        '';
+      };
+
+      name = mkOption {
+        type = types.str;
+        default = "f";
+        description = ''
+          Name of function used to call fff.
+        '';
+      };
+
+      commands = mkOption {
+        type = types.str;
+        default = ''
+          # cd to exited folder for bash and zsh
+          fff "$@"
+          cd "$(cat "''${XDG_CACHE_HOME:=''${HOME}/.cache}/fff/.fff_d")" 
```

I think this could be
```suggestion
          cd "$(cat "$FFF_CD_FILE")" 
```
instead? Similar with `fishCommands` below.